web/src/locales/en.json

```diff
@@ -1,18 +1,17 @@
 {
   "info": {
-    "title": "About the app",
-    "text": "The Electricity Maps app provides real-time insights on web and mobile for citizens, energy experts, NGOs and policymakers promoting a deeper understanding of energy transition challenges.",
-    "open-source-text": "This app is <a href='{{link}}' target='_blank'>Open Source</a> and powered by the community.",
+    "title": "About Electricity Maps",
+    "text": "Electricity Maps is an API platform providing real-time and predictive electricity signals allowing any device to reduce their cost and emissions by informing them about the best time to consume electricity.",
```

Comment:
Small tweak suggestions to the grammar for clarity:
> Electricity Maps is an API platform that provides real-time and predictive electricity signals, enabling devices to reduce costs and emissions by identifying the best times to consume electricity.

Reply:
Also, I wonder if we should link to the commercial website here (at least for the time being). "To find out more..." etc etce

Reply:
Or just link the words "Electricity Maps" to the commercial website?

Reply:
I'll try the link! I'm a little bit cautious to change the copy for this one in particular as it's copy pasta from else where (job posting I believe) and having them drift slightly might be confusing

Reply:
Fair enough!

Reply:
I think it looks great!
<img width="411" alt="image" src="https://github.com/user-attachments/assets/97c22cab-799e-43e2-94ce-df59e88c83db" />

Thanks for the suggestion!